src/link/object.c

```diff
@@ -331,9 +338,8 @@ obj_ReadOpenFile(FILE * pObjfile, char *tzObjectfile)
 	tzHeader[4] = 0;
 	if (strncmp(tzHeader, "RGB", 3) == 0) {
 		switch (tzHeader[3]) {
-		case '3':
-		case '4': // V4 supports OAM sections, but is otherwise identical
-			obj_ReadRGB(pObjfile);
+		case '5':
```

Comment:
I think @NieDzejkob means that we should declare something like `const char objectFileVersion = '5'` or `#define OBJECT_FILE_VERSION 5`.

This sounds fair enough to me, since at this point, the switch statement (implying multiple allowed versions) is a holdover from when the RGB0 format was supported, which required a different method of parsing.

Alternatively, we could use the whole string (so `const char *objectFileVersion = "RGB5"`).